dockerfiles/init/manifests/che.env

```diff
@@ -543,6 +543,9 @@ CHE_SINGLE_PORT=false
 #CHE_KEYCLOAK_CLIENT__ID=che-public
 #CHE_KEYCLOAK_ALLOWED__CLOCK__SKEW__SEC=3
 #CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD=true
+#CHE_KEYCLOAK_USE__NONCE=true
```

Comment:
properties should be documented as it's for user editing (while the properties file is quite hidden)

Reply:
documented in the `che.env` file or in the docs as a separate PR ?

Reply:
in che.env like https://github.com/eclipse/che/blob/6897a6e8b45c3084639d748edc802247a90a6bc9/dockerfiles/init/manifests/che.env#L18-L21 (based on what you've commented on properties file

Reply:
Added the comments in commit https://github.com/eclipse/che/pull/8650/commits/4ab66878b134a6e032b579d5d29479c1ff7b6c94

is it what you were expecting @benoitf  ?